static/UserDefault.json

```diff
@@ -3,7 +3,7 @@
     "LinkedAccounts": {},
     "Extensions": {
         "PeacockEscalations": {},
-        "PeacockFavoriteContracts": [],
```

Comment:
Favorite contracts have their own system, this will break it.

Reply:
Could you please elaborate?

Reply:
I added it back. Deleting it was an oversight.